config_example.json

```diff
@@ -1223,15 +1223,16 @@
   },
   {
    "name": "CoinbasePro",
```

Comment:
Can rename this to Coinbase and same in configtest.json, since we've go test coverage for the exchange name replacement now with the config upgrade